Makefile

```diff
@@ -88,7 +99,7 @@ uninstall:
 	@test -f "$(DEST)/bin/lima" || echo "lima not found in $(DEST) prefix"
 	rm -rf \
 		"$(DEST)/bin/lima" \
-		"$(DEST)/bin/limactl" \
+		"$(DEST)/bin/limactl$(exe)" \
 		"$(DEST)/bin/nerdctl.lima" \
 		"$(DEST)/bin/docker.lima" \
 		"$(DEST)/bin/podman.lima" \
```

Comment:
Probably we do not need this the wrapper scripts for Windows.

Reply:
I'm not sure it is related, but it is possible to replace both "lima" and "nerdctl.lima" with programs - if wanted.

Currently it is assumed that user has installed `sh.exe` (bash.exe) just as they have installed `ssh.exe` and so on...

Otherwise _all_ scripts would need to be rewritten, which seems unnecessary ?

Something like "Git for Windows" is enough to install, to get the requirements.